charts/linkerd-crds/templates/policy/authorization-policy.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     kind: AuthorizationPolicy
     plural: authorizationpolicies
     singular: authorizationpolicy
-    shortNames: []
+    shortNames: [ap, authzpolicy]
```

Comment:
Hey @javaducky sorry for the delay in reviewing this. Since these resource shortnames are global, I don't think we should add `ap` as an option because it's more likely to conflict with something else. Let's have AuthorizationPolicy's shortname only be `authzpolicy`.

After this change, this should be good to merge. Thanks again!

Reply:
@kleimkuhler no problem...I've pushed an update.